### ISSUE TITLE
Test suite fails: "failed to use local certificate chain"

### ISSUE THREAD

**Issue by eserte** [debug-ito/AnyEvent-WebSocket-Server#5]
The test suite fails on some of my smoker systems (e.g. linux debian 10):
```
# Testing AnyEvent::WebSocket::Server 0.09, Perl 5.033006, /opt/perl-5.33.6/bin/perl5.33.6
t/00-load.t ................. ok
    # No tests run!

#   Failed test 'No tests run for subtest "conn:wss, separate"'
#   at /home/cpansand/.cpan/build/2021012209/AnyEvent-WebSocket-Server-0.09-0/t/testlib/ConnConfig.pm line 119.
t/data/ssl_test.crt: failed to use local certificate chain (cert_file or cert) at /home/cpansand/.cpan/build/2021012209/AnyEvent-WebSocket-Server-0.09-0/blib/lib/AnyEvent/WebSocket/Server.pm line 63.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 2.
t/connetion_active_close.t .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/2 subtests 
    # No tests run!
... etc ...
```

I think this is because the RSA key used here is just of length 2024 bits, but newer Debian require at least 2048 bits.
See also https://www.debian.org/releases/buster/amd64/release-notes/ch-information.en.html#openssl-defaults
So generating a new test certificate could probably help here.

**Comment by debug-ito** [comment]
Thanks for reporting! I'll look at it.

**Comment by debug-ito** [comment]
Released AnyEvent-WebSocket-Server-0.10. It includes the RSA key with 3072 bit length. This should fix the issue.
